scripts/actions/abilities/pets/lunar_roar.lua

```diff
@@ -1,16 +1,24 @@
 -----------------------------------
--- Aerial Armor
+-- Lunar Roar
 -----------------------------------
 local abilityObject = {}
 
 abilityObject.onAbilityCheck = function(player, target, ability)
-    return 0, 0
+    return xi.job_utils.summoner.canUseBloodPact(player, player:getPet(), target, ability)
 end
 
-abilityObject.onPetAbility = function(target, pet, skill)
-    target:dispelStatusEffect()
-    target:dispelStatusEffect()
-    skill:setMsg(xi.msg.basic.NONE)
+abilityObject.onPetAbility = function(target, pet, petskill, summoner, action)
+    xi.job_utils.summoner.onUseBloodPact(target, petskill, summoner, action)
+
+    local effect = target:dispelStatusEffect()
+    if effect == xi.effect.NONE then
+        petskill:setMsg(xi.msg.basic.JA_NO_EFFECT_2)
+    else
+        target:dispelStatusEffect()
+        target:dispelStatusEffect()
+        petskill:setMsg(xi.msg.basic.NONE)
```

Comment:
Wouldn't this be dispelling 3 status effects (wiki says two effects) and then not reporting a dispel?

Reply:
Retail doesn't report when a effect is dispelled based on captures but I see what you mean. I can't remember if I intended to remove the additional dispel or left it from testing. I will have to report back on this

Reply:
I corrected the issue of the effects. This is a reference to retail with no display message of removal 
![image](https://github.com/LandSandBoat/server/assets/98081508/2814906d-bb7d-46dd-84a1-14ac103da191)

When nothing can be removed, the client is sent "no effect" message
![image](https://github.com/LandSandBoat/server/assets/98081508/bdf542e5-3413-49a6-a754-63b9b266ace5)

Reply:
Interesting. The only difference is the "reaction" other than the message but I don't know if that actually produces a different message to the client. Thanks for the caps.